store/landmarks.js

```diff
@@ -0,0 +1,39 @@
+
+//Action Constants
+const GOT_LANDMARKS = "GOT_LANDMARKS"
+const THREW_ERROR = "THREW_ERROR"
+
+//Action Creators
+const gotLandmarks = landmarks => ({type: GOT_LANDMARKS, landmarks})
+const threwError = err => ({type: THREW_ERROR, err})
```

Comment:
can we change this to something a little more descriptive? eg. `errorGettingLandmarks` or `dbThrewError`